packages/psss/src/routes/PageRoutes.js

```diff
@@ -4,23 +4,35 @@
  */
 
 import React from 'react';
-import { Redirect, Switch } from 'react-router-dom';
+import { Switch, Route } from 'react-router-dom';
 import { AlertsOutbreaksView } from '../views/AlertsOutbreaksView';
+import { AlertsOutbreaksSummaryView } from '../views/AlertsOutbreaksSummaryView';
 import { CountriesReportsView } from '../views/CountriesReportsView';
 import { CountryReportsView } from '../views/CountryReportsView';
 import { PrivateRoute } from './PrivateRoute';
+import { UnAuthorisedView } from '../views/UnauthorisedView';
+import { NotFoundView } from '../views/NotFoundView';
+import { checkIsAuthorisedForCountry, checkIsAuthorisedForMultiCountry } from '../utils/auth';
 
 export const PageRoutes = React.memo(() => (
   <Switch>
-    <PrivateRoute exact path="/">
+    <PrivateRoute exact path="/" authCheck={checkIsAuthorisedForMultiCountry}>
       <CountriesReportsView />
     </PrivateRoute>
-    <PrivateRoute path="/weekly-reports/:countryName">
+    <PrivateRoute path="/weekly-reports/:countryCode" authCheck={checkIsAuthorisedForCountry}>
       <CountryReportsView />
     </PrivateRoute>
-    <PrivateRoute path="/alerts">
+    <PrivateRoute path="/alerts/:countryCode" authCheck={checkIsAuthorisedForCountry}>
+      <AlertsOutbreaksSummaryView />
+    </PrivateRoute>
+    <PrivateRoute path="/alerts" authCheck={checkIsAuthorisedForMultiCountry}>
       <AlertsOutbreaksView />
     </PrivateRoute>
-    <Redirect to="/" />
```

Comment:
I replaced the redirect with a `NotFoundView` because `single country users` can not get redirected to the `/` route and would get an `UnauthorisedView` when they enter an invalid url. The other option would be to connect this to redux and use a dynamic route for the redirect so that `single country users` would get redirected to their actual home page instead. However I decided that in the end an `UnauthorisedView` was more informative.